### ISSUE TITLE
CI image has no OOTB way to configure app

### ISSUE THREAD

**Issue by bikingcoder** [dokku/ci-docker-image#24]
### Description of feature

New Dokku user here. I've just converted an internal web app to Dockerfile based Dokku deployment using Gitlab CI. I used the snippet suggested in the docs, and the provided image, but was surprised to find that there was no way for me to pass through some secret config vars from CI to my app. 

I ended up adding this script to the `/bin` dir of the official Dokku CI image (and installing BASH). Maybe I'm missing something as this seems like it would be a common requirement.

```
#!/bin/bash

# Adapted from dokku-deploy (https://github.com/dokku/ci-docker-image)

# Any var in the environment of the form DOKKU_VAR will set VAR
# using the Dokku config:set command.

set -e
set -o pipefail

setup-ssh

commit_sha="$(parse-ci-commit)"
app_name="$(parse-app-name)"
ssh_remote="ssh://dokku@$(parse-ssh-host):$(parse-ssh-port)"

# grep the names of the vars to set
vars=$(env | sed -n "s/^.*DOKKU_\(\S*\)=.*$/\1/p")

# build the args for config:set command
args=( )
for v in $vars;do
  n=DOKKU_$v
  args+=("${v}=${!n}")
done

ssh "$ssh_remote" -- config:set "$app_name" ${args[*]}
```

**Comment by mcgaw** [comment]
Hi. Yep, I realise there are a _ton_ of actions one could take. I was merely relating the experience of a  _day one_ Dokku user, with a standard web app deploy. That's all.
I've looked at the Ansible repo, and can see there is indeed a module to configure applications. I wasn't aware of it, and in fact this is only in the _advanced_ section of the docs. I don't think a day one user deploying a simple web app should be expected to go to the advanced section of the docs, do you?
Anyway, the snippet is there for anyone else. And Dokku is working beautifully!

**Comment by josegonzalez** [comment]
I implemented this in #28 as a file hook. I think that strikes the right balance between functionality and complexity of the implementation. Hopefully this is useful to you and others implementing similar pre-deploy configuration.
